### PR TITLE
Update azs-how-create-vm.md Line 58-60 and image

### PR DESCRIPTION
**Pull request by jakerenouf**

### CODE REVIEW

**Comment by jakerenouf** [comment]
Removed old image, and I am working in a different branch for the title updates - unsure why some where added to this pull request but should not happen again

File: articles/azure/azs-how-create-vm.md [hunk 1]
```diff
@@ -55,9 +55,9 @@ VMs provide the basic compute building blocks in Azure Stack. You can create VMs
 
     ![Create virtual machine > Size](images/azsp_createvm_size.png)
 
-7. In the *Settings* step, change any of the optional settings as required for your VM, then click **OK**.
+7. In the *Settings* step, change any of the optional settings as required for your VM, as of version 1811 for storage "managed disks" are selected by default and you need to select the public inbound ports you want enabled, then click **OK**.
```
Comment (BaileyLawson) [inline articles/azure/azs-how-create-vm.md]: Think this sentence may need rewording, or possibly splitting into two sentences.

Comment (jakerenouf) [reply articles/azure/azs-how-create-vm.md]: Ammended so that this adition shows as a note after the image

**Comment by jakerenouf** [comment]
Amended this and request #364 inline with the naming scheme

**Comment by jakerenouf** [comment]
Done in other PRs

**Pull request closed without merging**